src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +151,60 @@ function DisplayLine({
   )
 }
 
+function DollarLineAndAtLine({
+  line,
+  container = 'div'
+}: {
+  line: LineContainer
+  container?: string
+}): JSX.Element {
+  if (line.type === 'RulingDollarLine') {
+    const rulingDollarLine = line as RulingDollarLine
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
+      [
+        <hr
+          key={101}
+          className={`Transliteration__${line.type} ${rulingDollarLine.number}`}
+        />
+      ]
+    )
+  } else {
+    return React.createElement(
+      container,
+      { className: classNames([`Transliteration__ControlLine`]) },
+      [
+        <div key={101} className="Transliteration__DollarLineAndAtLine">
+          ({line.content[0].value.trimStart()})
+        </div>
+      ]
+    )
+  }
+}
+
+const lineComponents: ReadonlyMap<
+  string,
+  FunctionComponent<{
+    line: Line
+    container?: string
+  }>
+> = new Map([
+  ['$', DollarLineAndAtLine],
+  ['@', DollarLineAndAtLine]
+])
+
 export function Transliteration({
   text: { lines }
 }: {
   text: Text
 }): JSX.Element {
   return (
     <ol className="Transliteration">
-      {lines.map((line: Line, index: number) => (
-        <DisplayLine key={index} container="li" line={line} />
-      ))}
+      {lines.map((line: Line, index: number) => {
+        const LineComponent = lineComponents.get(line.prefix) || DisplayLine
```

Comment:
Do not use the `prefix` use `type`. The prefix will also be removed along with `content`.